ci/Dockerfile

```
@@ -1,62 +1,140 @@
-FROM a12e/docker-qt:5.14-gcc_64
+ARG QT_VERSION=5.15.2
+
+# QT Installation Image --------------------------------------------------------
+FROM ubuntu:18.04 AS qt-install
+
+ARG QT_VERSION
+
+RUN apt update && apt full-upgrade -y && apt install -y --no-install-recommends \
+    sudo python3 python3-pip \
```

Please keep the formatting consistent as in the rest of the file and use:
```suggestion
 && sudo python3 python3-pip \
```
See:
https://github.com/status-im/status-desktop/blob/fb28070e340bafc5258a6228dd66b2cfe326c8b0/ci/Dockerfile#L130-L136